conda/recipes/ucxx/recipe.yaml

```diff
@@ -0,0 +1,323 @@
+            - include/ucxx/worker_progress_thread.h
+            - lib/libucxx.so
+    about:
+      homepage: ${{ load_from_file("python/libucxx/pyproject.toml").project.urls.Homepage }}
+      license: ${{ load_from_file("python/libucxx/pyproject.toml").project.license.text }}
+      summary: ${{ load_from_file("python/libucxx/pyproject.toml").project.description }}
+
+  - package:
+      name: libucxx-examples
+      version: ${{ version }}
+    build:
+      string: ${{ date_string }}_${{ head_rev }}
+      script: cmake --install cpp/build --component examples
+      dynamic_linking:
+        overlinking_behavior: "error"
+    requirements:
+      build:
+        - ${{ compiler("c") }}
+        - ${{ compiler("cxx") }}
+        - ${{ compiler("cuda") }}
+        - cmake ${{ cmake_version }}
+      host:
+        - libucxx
+      run:
+        - ${{ pin_subpackage("libucxx", exact=True) }}
+      ignore_run_exports:
+        from_package:
+          - cuda-cudart-dev
+          - python
+          - ucx
+        by_name:
+          - cuda-version
+          - if: cuda_major == "11"
+            then: cudatoolkit
+    about:
+      homepage: ${{ load_from_file("python/libucxx/pyproject.toml").project.urls.Homepage }}
+      license: ${{ load_from_file("python/libucxx/pyproject.toml").project.license.text }}
+      summary: libucxx examples executables
+
+  - package:
+      name: libucxx-tests
+      version: ${{ version }}
+    build:
+      string: cuda${{ cuda_major }}_${{ date_string }}_${{ head_rev }}
+      script: cmake --install cpp/build --component testing
+      dynamic_linking:
+        overlinking_behavior: "error"
+    requirements:
+      build:
+        - ${{ compiler("c") }}
+        - ${{ compiler("cxx") }}
+        - ${{ compiler("cuda") }}
+        - cmake ${{ cmake_version }}
+      host:
+        - cuda-version =${{ cuda_version }}
+        - if: cuda_major == "11"
+          then: cudatoolkit
+          else: cuda-cudart-dev
+        - libucxx
+      run:
+        - if: cuda_major == "11"
+          then: cudatoolkit
+        - ${{ pin_compatible("cuda-version", upper_bound="x", lower_bound="x") }}
+        - ${{ pin_subpackage("libucxx", exact=True) }}
+      ignore_run_exports:
+        from_package:
+          - cuda-cudart-dev
+          - python
+          - ucx
+        by_name:
+          - cuda-version
+          - if: cuda_major == "11"
+            then: cudatoolkit
+    about:
+      homepage: ${{ load_from_file("python/libucxx/pyproject.toml").project.urls.Homepage }}
+      license: ${{ load_from_file("python/libucxx/pyproject.toml").project.license.text }}
+      summary: libucxx tests executables
+
+  - package:
+      name: ucxx
+      version: ${{ version }}
+    build:
+      string: cuda${{ cuda_major }}_py${{ py_version }}_${{ date_string }}_${{ head_rev }}
+      dynamic_linking:
+        overlinking_behavior: "error"
+        missing_dso_allowlist:
+          - "lib/libpython3.12.so*"
```

Comment:
iirc, I think we expect this to link to Python. We shouldn't ignore this. This could be indicating an issue with our use of variant builds for each Python version?